tests/data_context/fixtures/post_init_project_v0.8.0_A/great_expectations/great_expectations.yml

```diff
@@ -68,13 +68,7 @@ stores:
 #      key_prefix:
 
   evaluation_parameter_store:
-    module_name: great_expectations.data_context.store
-    class_name: EvaluationParameterStore
-
-  local_site_html_store:
-    module_name: great_expectations.data_context.store
-    class_name: HtmlSiteStore
-    base_directory: uncommitted/data_docs/local_site/
```

Comment:
Nitpick: There are a pile of other fixture templates that _might_ have keys that could be removed.